### PR TITLE
{Flake8} Ignore W503 and E203

### PR DESCRIPTION
**Pull request by StrawnSC**
**Description**

Removes the following Flake8 rules because they are not compliant with PEP 8 (the official python style guide):
[Line break occurred before binary operator (W503)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.flake8rules.com%2Frules%2FW503.html&data=05%7C01%7Csilasstrawn%40microsoft.com%7Ceefded93d3674988891f08da1cd00efe%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637853977221702278%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=xghtbRXJ6WTwx76L0fzHNpCia55E76wnEcl3V0AQ2dc%3D&reserved=0) (directly violates PEP8)
[Whitespace before ‘:’ (E203)](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fwww.flake8rules.com%2Frules%2FE203.html&data=05%7C01%7Csilasstrawn%40microsoft.com%7Ceefded93d3674988891f08da1cd00efe%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637853977221702278%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=KcLWJ%2F5m01k2wC%2FiXG8nkdvi%2FhFgoZNz6RuCtNz8h9w%3D&reserved=0) (violates PEP8 when the colon is used as a binary operator in a list slice)

This would allow CLI devs to use the [black](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fpsf%2Fblack&data=05%7C01%7Csilasstrawn%40microsoft.com%7Ceefded93d3674988891f08da1cd00efe%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C637853977221702278%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=yiLT5hs3wvASsYjnfSxpsnVm01jPZPbfVlI2r0wnopY%3D&reserved=0) code formatter

**Testing Guide**

**History Notes**

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Review by wangzelin007 — APPROVED** [review]

**Comment by wangzelin007** [comment]
LGTM

File: .flake8 [hunk 1]
```diff
@@ -8,6 +8,8 @@ ignore =
     F811,  # redefinition of unused, to be removed in the future
     C901   # code flow is too complex, too many violations, to be removed in the future
     W504   # line break after binary operator effect on readability is subjective
+    W503   # line break before binary operator, not compliant with PEP 8
+    E203   # whitespace before ':', not compliant with PEP 8
```
Comment (jiasli) [inline .flake8:12]: Could you give an example that violates E203?

**Comment by jiasli** [comment]
Comment from @StrawnSC:

> We originally hoped this change would allow us to use the code formatter black, but unfortunately black turned out to not be very useful for passing the style check. When I first tested out black, I did so on code that already passed the style check. Sadly it didn't help much on code that failed the style check

**Pull request closed without merging**